file-format-types.rdf

```diff
@@ -0,0 +1,202 @@
+  </udfrs:GenreFacetType>
+  <udfrs:GenreFacetType rdf:ID="PageDescription">
+    <rdfs:isDefinedBy rdf:resource="http://pcdm.org/file-format-types#"/>
+    <rdfs:subClassOf rdf:resource="Text" />
+    <skos:exactMatch rdf:resource="http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#PaginatedTextDocument" />
+    <skos:exactMatch rdf:resource="http://reference.data.gov.uk/technical-registry/PageDescription" />
+    <skos:exactMatch rdf:resource="http://www.udfr.org/onto#PageDescriptionGenre" />
+  </udfrs:GenreFacetType>
+  <udfrs:GenreFacetType rdf:ID="UnstructuredText">
+    <rdfs:isDefinedBy rdf:resource="http://pcdm.org/file-format-types#"/>
+    <rdfs:subClassOf rdf:resource="Text" />
+    <skos:closeMatch rdf:resource="http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#PlainTextDocument" />
+    <skos:exactMatch rdf:resource="http://reference.data.gov.uk/technical-registry/UnstructuredText" />
+  </udfrs:GenreFacetType>
+  <udfrs:GenreFacetType rdf:ID="StructuredText">
+    <rdfs:isDefinedBy rdf:resource="http://pcdm.org/file-format-types#"/>
+    <rdfs:subClassOf rdf:resource="Text" />
+    <skos:exactMatch rdf:resource="http://reference.data.gov.uk/technical-registry/StructuredText" />
+  </udfrs:GenreFacetType>
+  <!-- Structured text genres -->
+  <udfrs:GenreFacetType rdf:ID="SourceCode">
+    <rdfs:isDefinedBy rdf:resource="http://pcdm.org/file-format-types#"/>
+    <rdfs:subClassOf rdf:resource="StructuredText" />
+    <skos:exactMatch rdf:resource="http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#SourceCode" />
+  </udfrs:GenreFacetType>
+  <udfrs:GenreFacetType rdf:ID="Markup">
+    <rdfs:isDefinedBy rdf:resource="http://pcdm.org/file-format-types#"/>
+    <rdfs:subClassOf rdf:resource="StructuredText" />
+    <skos:exactMatch rdf:resource="http://reference.data.gov.uk/technical-registry/MarkupText" />
+  </udfrs:GenreFacetType>
+  <udfrs:GenreFacetType rdf:ID="HTML">
+    <rdfs:isDefinedBy rdf:resource="http://pcdm.org/file-format-types#"/>
+    <rdfs:subClassOf rdf:resource="Markup" />
+    <skos:exactMatch rdf:resource="http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#HTMLDocument" />
+    <skos:exactMatch rdf:resource="http://vocab.getty.edu/aat/300266021" />
+  </udfrs:GenreFacetType>
+  <udfrs:GenreFacetType rdf:ID="Presentation">
+    <rdfs:isDefinedBy rdf:resource="http://pcdm.org/file-format-types#"/>
+    <rdfs:subClassOf rdf:resource="Document" />
+    <skos:exactMatch rdf:resource="http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#Presentation" />
+    <skos:exactMatch rdf:resource="http://reference.data.gov.uk/technical-registry/Presentation" />
+    <skos:exactMatch rdf:resource="http://www.udfr.org/onto#PresentationGenre" />
+  </udfrs:GenreFacetType>
+  <udfrs:GenreFacetType rdf:ID="Spreadsheet">
+    <rdfs:isDefinedBy rdf:resource="http://pcdm.org/file-format-types#"/>
+    <rdfs:subClassOf rdf:resource="Document" />
```

Comment:
I think that's definitely true of nfo:PaginatedTextDocument -- but I don't agree about pronom:PageDescription or udfr:PageDescriptionGenre.  Pronom has a separate WordprocessedText class, which seems to pretty clearly indicate that it's separate.  There's not a lot to go on for UDFR since there really aren't any descriptions, examples or relations defined, but my reading of udfr:DocumentGenre is that it's more like pronom:WordprocessedText, too (distinct from the other genres, not a superclass of them).